packages/launchpad/cypress/e2e/top-nav-launchpad.cy.ts

```diff
@@ -504,7 +504,7 @@ describe('Launchpad Top Nav Workflows', () => {
             cy.findByRole('button', { name: 'Log in' }).click()
           })
 
-          cy.findByRole('dialog', { name: 'Log in to Cypress' }).within(() => {
+          cy.findByRole('dialog').within(() => {
```

Comment:
After the first button click, the dialog no longer has the name "Log in to Cypress" - it's turned into an error popup. Because we now re-query the DOM though, .findByRole() failed to find any dialog with a matching name, and the test failed.

We're now within *a dialog*, rather than within *a dialog named Log In*. This PR forced us as test-writers to be clearer about exactly what we want - IMO a good side effect, but unfortunately a breaking change.

Reply:
Will the be an obvious change to the cy.findByRole behavior for users upgrading? I wouldn't have expected this to be broken, even after reviewing these changes 😬

Reply:
Same - after debugging I could see why it was failing, but it's not something I *expected* either. :grimacing: 

I don't think it's terribly obvious, but I also don't think it's avoidable. This is just the cost of fixing detached dom errors - if your DOM changes, Cypress will notice the change.